src/ripple/protocol/STObject.h

```diff
@@ -401,7 +401,7 @@ class STObject
         if (!cf)
             throw std::runtime_error ("Wrong field type");
 
-        cf->setValue (value);
+        cf->setValue (std::move (value));
```

Comment:
@seelabs:  Interesting.  I'll need to do some investigating to see why the inferred `V` behaves differently from the explicit `V`.  I still have plenty to learn about the inner workings of templates.